src/views/app.vue

```diff
@@ -38,6 +41,17 @@ export default {
     }
   },
   mounted: async function () {
+    if (navigator.userAgent.match(/Android/i) ||
```

Comment:
Let's not exclude tablet users. I think we should make this dependent on screen size and not device type/os.